packages/components/src/date-time/index.js

```diff
@@ -35,7 +35,7 @@ export class DateTimePicker extends Component {
 
 	render() {
 		const {
-			currentDate,
+			currentDate = new Date(),
```

Comment:
I'm not sure this default is needed, as it's initialised elsewhere, and should probably be handled by default props instead. This is a reminder to me to remove it!